### PR TITLE
Snapping it!

### PR DESCRIPTION
**Pull request by cprov**
Highly tentative changes in order to snap the service and simplify the deployment.

### CODE REVIEW

**Comment by earnubs** [comment]
The failures don't look related to the pr, I think they're because nock isn't being cleanup up after a test.

```
File [/home/travis/build/canonical-ols/javan-rhino/src/components/payments-form/countries.js] ignored, nothing could be mapped
```

Not sure about that one.

File: snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  # Service ... broken still.
+  #javan-rhino:
+  #  command: production >$SNAP_DATA/server.log 2>&1
+  #  daemon: simple
+  #  plugs: [network, network-bind]
+
+parts:
+  javan-rhino:
+    plugin: nodejs
+    node-engine: '6.5.0'
```
Comment (bartaz) [inline snapcraft.yaml]: @earnubs I guess we were moving back to 4 due to some issues with auth libraries?

Comment (earnubs) [inline snapcraft.yaml]: @bartaz now resolved upstream https://github.com/havard/node-openid/commit/59c7d23549fbfd45a84dabca4beb4f61b9f5f35a, so we'll be bumping back up to 6.

**Comment by earnubs** [comment]
We're dropping mongodb for node-client-sessions

**Comment by earnubs** [comment]
Closing, no activity in a month, please reopen when ready.

**Pull request closed without merging**